python/ray/tests/test_state_api.py

```diff
@@ -1719,37 +1701,6 @@ async def test_state_data_source_client_limit_distributed_sources(ray_start_clus
         port = int(node["NodeManagerPort"])
         client.register_raylet_client(node_id, ip, port)
 
-    """
-    Test tasks
-    """
-
-    @ray.remote
-    def long_running():
-        import time
-
-        time.sleep(300)
-
-    @ray.remote
-    def f():
-        ray.get([long_running.remote() for _ in range(2)])
-
-    # Driver: 2 * f
-    # Each worker: 2 * long_running
-    # -> 2 * f + 4 * long_running
-
-    refs = [f.remote() for _ in range(2)]  # noqa
-
-    async def verify():
-        result = await client.get_task_info(node_id, limit=2)
```

Comment:
This function is no longer used, it's replaced by `get_all_task_info` which queries the task backend directly.